contracts/exchange/test/core.ts

```diff
@@ -749,30 +772,20 @@ describe('Exchange core', () => {
             return expect(tx).to.revertWith(expectedError);
         });
 
-        it('should revert if makerAssetAmount is 0', async () => {
+        it('should noop if makerAssetAmount is 0', async () => {
             signedOrder = await orderFactory.newSignedOrderAsync({
                 makerAssetAmount: new BigNumber(0),
             });
-            const orderHash = orderHashUtils.getOrderHashHex(signedOrder);
-            const expectedError = new ExchangeRevertErrors.OrderStatusError(
-                orderHash,
-                OrderStatus.InvalidMakerAssetAmount,
-            );
-            const tx = exchangeWrapper.cancelOrderAsync(signedOrder, makerAddress);
-            return expect(tx).to.revertWith(expectedError);
+            const tx = await exchangeWrapper.cancelOrderAsync(signedOrder, makerAddress);
```

Comment:
Will the length of the logs be zero if the tx reverts? If so, we should probably also assert that the tx succeeds.

Reply:
The transaction will throw an error if it reverts rather than returning a txReceipt.

Reply:
`.fulfilled('')`